src/Documentation/clusters_and_clients/configuration_guide/typical_configurations.md

```diff
@@ -126,7 +126,7 @@ var gateways = new IPEndPoint[]
     new IPEndPoint(OTHER_SILO__IP_ADDRESS_N, 11111),
 };
 var client = new ClientBuilder()
-    .UseStaticClustering(gateways)
+    .UseStaticClustering(gatewayPort: 30000)
```

Comment:
The way it is already (prior to this change) looks correct to me. The change only works if the method is changed to `UseLocalhostClustering(gatewayPort: 30000)`, since `UseStaticClustering` accept a collection of `IPEndPoint` (IP address + port). That's only applicable for when the silo is deployed to the local host.